compiler/parser/parser.peg

```diff
@@ -580,12 +583,15 @@ MergeOp
     }
 
 OverOp
-  = "over" _ exprs:Exprs locals:Locals? scope:Scope? {
-      RETURN(MAP("kind": "Over", "exprs": exprs, "locals": locals, "scope": scope))
+  = "over" _ exprs:Exprs locals:Locals? body:Lateral? {
+      RETURN(MAP("kind": "Over", "exprs": exprs, "locals": locals, "body": body))
     }
 
-Scope
-  = __ "=>" __ "(" __ seq:Sequential __ ")" { RETURN(seq) }
+// This is a little weird that you can have double parens, e.g., => ((foo | count()))
+// But we need the parens to define the subquery boundaries. Not sure how to fix it.
```

Comment:
Are you sure this is true?

```console
$ go run ./cmd/zc -C 'over a => ((b | c))'
error parsing Zed at column 15:
over a => ((b | c))
          === ^ ===
exit status 1
```

Reply:
It's true for `over a => ((count()))`

How about we make an issue to  work on this some more?  All the tests are passing.

Reply:
Sounds good.

Reply:
Hmm, you're right that the comment was wrong but I still don't understand why the double-parent count parses.